docs/docusaurus/docs/guides/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_missingness_data_assistant.md

```diff
@@ -0,0 +1,146 @@
+:::
+
+## Prepare a new Expectation Suite
+
+Run the following code to prepare a new Expectation Suite with the Data Context `add_expectation_suite(...)` method:
+
+```python name="tests/integration/docusaurus/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_missingness_data_assistant.py expectation_suite"
+```
+
+## Run the Missingness Data Assistant
+
+To run a Data Assistant, you can call the `run(...)` method for the assistant. However, there are numerous parameters available for the `run(...)` method of the Missingness Data Assistant. For instance, the `exclude_column_names` parameter allows you to define the columns that should not be Profiled.
+
+1. Run the following code to define the columns to exclude:
+
+  ```python name="tests/integration/docusaurus/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_missingness_data_assistant.py exclude_column_names"
+  ```
+
+2. Run the following code to run the Missingness Data Assistant:
+
+  ```python name="tests/integration/docusaurus/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_missingness_data_assistant.py data_assistant_result"
+  ```
+
+  In this example, `context` is your Data Context instance.
+
+  :::note
+  If you consider your `BatchRequest` data valid, and want to produce Expectations with ranges that are identical to the data in the `BatchRequest`, you don't need to alter the example code. You're using the default `estimation` parameter (`"exact"`). To identify potential outliers in your `BatchRequest` data, pass `estimation="flag_outliers"` to the `run(...)` method.
+  :::
+
+  :::note
+  The Missingness Data Assistant `run(...)` method can accept other parameters in addition to `exclude_column_names` such as `include_column_names`, `include_column_name_suffixes`, and `cardinality_limit_mode`. To view the available parameters, see [this information](https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/rule_based_profiler/data_assistant/column_value_missing_data_assistant.py#L44).
+  :::
+
+## Save your Expectation Suite
+
+1. After executing the Missingness Data Assistant's `run(...)` method and generating Expectations for your data, run the following code to load and save them into your Expectation Suite:
+
+  ```python name="tests/integration/docusaurus/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_missingness_data_assistant.py get_expectation_suite"
+  ```
+
+2. Run the following code to save the Expectation Suite:
+
+  ```python name="tests/integration/docusaurus/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_missingness_data_assistant.py save_expectation_suite"
+  ```
+
+## Test your Expectation Suite
+
+1. Run the following code to use a `SimpleCheckpoint` to operate with the Expectation Suite and Batch Request that you defined:
```

Comment:
The `SimpleCheckpoint` is being deprecated.  Can you update this to use a regular Checkpoint with the Data Context's `.add_or_update_checkpoint(...)` method?  You'll also have to update the other references to the `SimpleCheckpoint` in this doc and in your example script.